projects/plugins/jetpack/modules/custom-css/custom-css/js/core-customizer-css.core-4.9.js

```diff
@@ -57,6 +57,14 @@
 			}).prependTo( '#css-help-links' );
 		}
 
+		// Show deprecation warning if Start Fresh option is enabled
+		if ( !! window._jp_css_settings.startFresh ) {
```

Comment:
If we don't want the notice showing in the customizer, I wonder if something like the following could work here. I've tested it on an Atomic test site and it is working for me, to hide the notice:

```suggestion
		if ( !! window._jp_css_settings.startFresh && window._currentSiteType !== 'atomic' ) {
```